front/tests/pages/operational-studies/paced-train-section.ts

```diff
@@ -202,6 +202,20 @@ class PacedTrainSection extends CommonPage {
     await this.verifyItemsVisibility(occurrenceIndex, 'occurrence');
   }
 
+  async clickOnOccurrence(pacedTrainIndex: number, occurrenceIndex: number) {
```

Comment:
To make the test more convenient
```suggestion
  async clickOnOccurrence({
    pacedTrainIndex,
    occurrenceIndex,
  }: {
    pacedTrainIndex: number;
    occurrenceIndex: number;
  }) {
```

Reply:
Done